src/OrbitGl/App.h

```diff
@@ -554,6 +554,11 @@ class OrbitApp final : public DataViewFactory,
   // no particular order.
   orbit_base::Future<std::vector<ErrorMessageOr<orbit_base::CanceledOr<void>>>> LoadAllSymbols();
 
+  // Automatically add a default Frame Track. It will choose the function
+  // yeti::internal::vulkan::(anonymous namespace)::QueuePresentKHR from the module `ggpvlk.so` if
```

Comment:
I think this comment is unnecessary and potentially problematic as it can become outdated. If we add fallbacks, other platforms, you need to keep this comment up-to-date.